code/game/objects/items/etherealdiscoball.dm

```diff
@@ -47,7 +48,7 @@
 
 /obj/structure/etherealball/proc/TurnOff()
 	TurnedOn = FALSE
-	set_light(0)
+	set_light(0, update = FALSE) // monkestation edit: use overlay lighting
```

Comment:
also, `update = FALSE` is needed to avoid calling `update_light`, which runtimes if the lighting system isn't `COMPLEX_LIGHT`

however, `set_light` itself *still works*, as overlay lighting registers the signals sent whenever light color/power/range/etc is changed